src/pages/home/report/ReportActionCompose.js

```diff
@@ -147,7 +147,7 @@ class ReportActionCompose extends React.Component {
         this.addEmojiToTextBox = this.addEmojiToTextBox.bind(this);
         this.focus = this.focus.bind(this);
         this.comment = props.comment;
-        this.shouldFocusInputOnScreenFocus = canFocusInputOnScreenFocus();
+        this.shouldFocusInputOnScreenFocus = !canFocusInputOnScreenFocus();
```

Comment:
Why this change is needed?

Reply:
need to autoFocus on devices that are supported as discussed in the issue.

Reply:
discussion link, please ?

Reply:
https://github.com/Expensify/App/issues/4678#issuecomment-904413983

Reply:
>  by default keyboard should be open (auto-focused) because the user is here to start the chat.

And I think the same. The keyboard should open when it's a new chat and you see this avatar preview and in all other cases, the keyboard should not open. Thus, this change is going to create a couple of regressions. 

Instead, only focus when the chat is new.